files/default/neutron-ha-tool.py

```diff
@@ -248,6 +248,14 @@ def run(args):
     return 1 if errors > 0 else 0
 
 
+class RandomAgentPicker(object):
+    def __init__(self, agents):
+        self.agents = agents
+
+    def pick(self):
```

Comment:
If so, probably `choose` is better than `choice` since it should be a verb.  But if you change it, would be better to stay consistent and rename the class to `RandomAgentChooser`.  Personally I slightly prefer `Picker` but there's not much difference.